### PR TITLE
docs(components): Update InputValidation docs

### PR DESCRIPTION
**Pull request by taylorvnoj**
## Motivations

InputValidation needs to follow new documentation structure

### Added

 
In the export for args, I made `message: ""` because otherwise, in the Controls section, there would be an error about not seeing/modifying props.  `validationMessages` can't go there either because it would be out of scope.

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: docs/components/InputValidation/InputValidation.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { useState } from "react";
```
Comment (rodrigoeidelvein) [inline docs/components/InputValidation/InputValidation.stories.mdx:2]: You can remove some of these imports: `ArgsTable`, `Canvas`, `Story` and `useState` are not being used

**Review by rodrigoeidelvein — CHANGES_REQUESTED** [review]
Looks good! Just need a small cleanup and it's good to go!

File: docs/components/InputValidation/InputValidation.stories.mdx (hunk 1 above)
Comment (taylorvnoj) [reply docs/components/InputValidation/InputValidation.stories.mdx]: Oops! Thanks for catching that!

**Review by rodrigoeidelvein — APPROVED** [review]
LGTM!

**Pull request merged**